### ISSUE TITLE
interpret_hurst uses hard-coded confidence interval irrespective of sample size

### ISSUE THREAD

**Issue by phelps-sg** [anabugaenko/hurstexponent#6]
The function `utils.interpret_hurst` returns "Perfect diffusivity: .." if the
estimated Hurst exponent is 0.05 +- 0.01.  This confidence interval is hard-coded in the call to round:

~~~python
H = round(H, 2)
~~~

However, the correct interval will depend on the required confidence level (e.g. p<0.05) as well as the sample size.  

Similarly for H > 0.5 and H < 0.5.  For example, if the estimated Hurst is 0.46 this might equally likely to be super-diffusive rather than sub-diffusive depending on the length of the series. 

In order to report meaningful results, a bootstrap should be used to estimate confidence intervals or perform statistical hypothesis testing for the required confidence level. 

Also, I do not think we can give a unique definitive interpretation for the case H == 0.5, since strictly speaking there is zero probability that $H = 0.5$ exactly, given a finite sample.   So instead of "interpreting" a Hurst estimate, it would make more sense to have a function that tells us the p-value for the null hypotheses $H < 0.5$ or $H > 0.5$.

**Comment by anabugaenko** [comment]
Rounding has been removed.
For the 0.5 case, I believe we can use approximate equality similar to `if np.isclose(H, 0.5):`

**Comment by phelps-sg** [comment]
>For the 0.5 case, I believe we can use approximate equality similar to if np.isclose(H, 0.5):
This is incorrect.  The result returned by the estimator is a point estimate whereas the "interpretation" refers to an assertion about the underlying data generation process.  

Yes, if were interpreting a true known parameter then you could use np.isclose(), but in the repo you are applying the interpretation to point estimates, which are themselves subject to inherent uncertainty caused by their sampling distribution.  You *cannot* assert that a point estimate of 0.5 corresponds to a data generation process with perfect diffusivity since that point estimate might in fact have been generated from estimating against an anti-persistent process.
